android/cli/commands/_build.js

```diff
@@ -1640,13 +1640,23 @@ process.exit(1);
 							}
 							if (missing) {
 								dependency.depended = module;
+
+								// attempt to include missing dependency
+								this.cli.tiapp.modules.push({
+									id: dependency.id,
+									version: dependency.version,
+									platform: ['android'],
```

Comment:
Do dependencies always have to be explicitly the same platform and native, or could it be commonjs too?
Do we need to specify the apiversion here to be the same as the requirer/SDK, or is that handled by validateTiModules?

Reply:
- commonjs modules aren't supported as module dependencies
- I tried my best to re-use our current module handling code, so all of the existing module checks will take place. `apiversion` needs to match the SDK
- It will be the same as our current module version handling code (I believe it's exact match only?)